newIDE/app/src/InstancesEditor/SelectedInstances.js

```diff
@@ -242,54 +250,39 @@ export default class SelectedInstances {
       this.rectanglesContainer.removeChild(this.selectedRectangles.pop());
     }
 
-    //Position the resize button.
     const show = selection.length !== 0;
-    const resizeButtonPos = this.toCanvasCoordinates(x2, y2);
-    resizeButtonPos[0] += buttonPadding;
-    resizeButtonPos[1] += buttonPadding;
 
-    const rightResizeButtonPos = this.toCanvasCoordinates(
-      x2,
-      y1 + (y2 - y1) / 2
-    );
-    rightResizeButtonPos[0] += buttonPadding;
-    rightResizeButtonPos[1] -= smallButtonSize / 2;
+    //Position the resize button.
```

Comment:
Much nicer now with all the buttons :)
Update the comment by the way: 
```suggestion
    // Position the resize buttons.
```